src/App.jsx

```diff
@@ -14,33 +15,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [activeSort, setActiveSort] = useState(null);
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const isOriginalOrder = () => {
+    return JSON.stringify(goods) === JSON.stringify(goodsFromServer);
```

Comment:
The `isOriginalOrder` function uses `JSON.stringify` to compare arrays. While this works, it can be inefficient for large arrays. Consider using a more efficient method, such as checking each element individually, if performance becomes an issue.